### DISCUSSION TITLE
comparing ant build tasks to gradle build tasks

### DISCUSSION THREAD

**Discussion by nyemiks** [opencrx/opencrx-documentation#32] [Q&A] [answered]
In opencrx version 4.1 for the custom project my build process was
1. ant clean.
2. ant build.
3. ant deliverables.
4. ant assemble.

Kindly confirm the proper sequence when performing the same using gradle on version 5.1
1. gradle clean.
2. gradle build.
3. gradle deliverables.
4. gradle assemble.

**Comment by christoph-mueller-crixp** [comment, marked as answer]
1. gradle clean
2. gradle assemble
3. gradle eclipse

if the build fails, try to add the option 
 --refresh-dependencies

you don't need step 3 unless you intend to build the project files for Eclipse

hope that helps

**Reply by nyemiks** [reply]
Noted. Thanks.
